lightwood/encoders/time_series/helpers/rnn_helpers.py

```diff
@@ -0,0 +1,54 @@
+
+
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
+
+import numpy as np
+
+def tensor_from_series(series):
+    return torch.tensor(series, dtype=torch.float, device=device).view(-1, 1, 1, 1).float()
+
+
+class DecoderRNNNumerical(nn.Module):
+    def __init__(self, hidden_size, output_size):
+        super(DecoderRNNNumerical, self).__init__()
+        self.hidden_size = hidden_size
+        self.gru = nn.GRU(output_size, hidden_size)
+        self.out = nn.Linear(hidden_size, output_size)
+
+
+    def forward(self, input, hidden):
```

Comment:
Hmh, thinking about it, does it even make sense to have a DecoderRNN at all ? As in, wouldn't the "decoding" part be just the `Linear` layer of the EncoderRNN that we train.

Reply:
it depends on what we define for ts as target

Reply:
separate PR, see previous questions